drivers/node/node.go

```diff
@@ -27,6 +28,13 @@ const (
 	Directory FindType = "d"
 )
 
+// StoragePool is the storage pool structure on the node
+type StoragePool struct {
+	*api.StoragePool
+	InitialSize  uint64
```

Comment:
just add comments above the variables to indicate what they are.

For e.g

// WorkloadSize is the size in GiB of the workload that will be launched by test on this storage pool